users/views.py

```diff
@@ -1,3 +1,33 @@
-from django.shortcuts import render
+import requests, jwt
 
-# Create your views here.
+from django.http.response import JsonResponse
+from django.views         import View
+
+from users.models         import User
+from django.conf          import settings
+
+class KakaoSignInView(View):
+    def get(self, request):
+        try:
+            access_token   = request.headers.get('Authorization', None)
+            
+            kakao_response = requests.get('https://kapi.kakao.com/v2/user/me', \
+                headers={'Authorization': f'Bearer {access_token}'})
+            
+            kakao_id       = kakao_response.json().get('id')
+            email          = kakao_response.json().get('kakao_account').get('email')
+            nickname       = kakao_response.json().get('properties').get('nickname')
+
+            obj, created = User.objects.get_or_create(
+                kakao_id = kakao_id,
+                email    = email,
+                name     = nickname
+            )
+            
+            access_token = jwt.encode({"id" : obj.id}, \
+                settings.SECRET_KEY, algorithm = settings.ALGORITHM)
+            
+            return JsonResponse({'message' : 'SUCCESS', 'ACCESS_TOKEN': access_token}, status=200)
```

Comment:
json body 의 key 값을 소문자로 통일해주세요.
```suggestion
            return JsonResponse({'message' : 'SUCCESS', 'access_token': access_token}, status=200)
```

Reply:
죄송합니다. 좀더 꼼꼼히 확인하겠습니다!